docker/traefik.override.yml

```diff
@@ -1,36 +1,75 @@
-version: '3.8'
-
 services:
```

Comment:
would you mind optimizing this docker file?
Ideally I would expect a base `traefik.yml` override with what is shared between the 2 cases.
Then 2 different `traefik_prod.yml` and `traefix_local.yml` with what they need to add from the base.
That would help us a lot in maintaining it over time.

Reply:
Sure, will do.

Reply:
Hey @mlodic 

I have split the configuration files into traefik.yml, traefik.override.prod.yml and traefik.override.local.yml as you wished...

I'm not sure if there are really benefits like this... but in any other way it's not possible, because for example the `command` key doesn't get merged, but rather it just takes the values of the last file provided.

Reply:
And sorry for changing the naming and not squashing... I was in a hurry. I can fix that, if you need me to.

Reply:
Hey @mlodic 

I have fixed the naming and the squashed the commits - could you please review? No rush, just a friendly heads-up. :)